packages/validator-bonds-cli-institutional/README.md

```diff
@@ -0,0 +1,277 @@
+- **Must be activating or activated**.
+
+
+```sh
+# Create a random keypair for a stake account to be created and funded to bond
+# The Validator Bonds program does not preserve stake account public keys as it merges and splits them
+solana-keygen new -o /tmp/stake-account-keypair.json
+
+# Creating a stake account. The SOLs will be funded to the Bond
+solana create-stake-account <stake-account-keypair> <Amount of SOL 1 for every 10,000 staked>
+
+# To couple the created stake account with the vote account
+# This causes the stake account to be in the Activating state.
+solana delegate-stake <stake-account-pubkey> <vote-account-address>
+
+# Funding Bond by assigning the stake account with the SOL amount in it
+validator-bonds-institutional -um fund-bond <bond-or-vote-account-address> \
+  --stake-account <stake-account-address> \
+  --stake-authority <withdrawer-stake-account-authority-keypair>
+```
+
+The meanings of parameters are as follows:
+
+- `<bond-or-vote-account-address>`: bond account that will be funded by the amount of
+  lamports from the stake account.
+- `--stake-account`: address of the stake account that will be assigned under the bonds program.
+- `--stake-authority`: signature of the stake account authority (probably withdrawer)
+  that permits to change the stake account authorities
+
+#### How to add more funds under the bond?
+
+It's as simple as creating a new stake account and funding it into the bond program.
+The amounts of SOLs delegated to the same validator are summed together.
+The validator bonds program may merge or split the accounts delegated to the same validator.
+It's not guaranteed to maintain the same stake accounts in the bond,
+but the amount of SOLs is always associated with the validator.
+
+### Show the bond account
+
+```sh
+validator-bonds-institutional -um show-bond <bond-or-vote-account-address>
+```
+
+To display all details about the Bond use `--with-funding` and `--verbose` parameters.
+Gathering all the details require multiple calls to RPC node and does not work
+properly with public RPC node (moniker `-um` or `--rpc-url mainnet-beta`).
+Use some other RPC node url as described at https://solana.com/rpc.
+
+```sh
+RPC_URL=<url-to-solana-rpc-node>
+validator-bonds-institutional -u$RPC_URL show-bond <bond-or-vote-account-address> \
+  --with-funding --verbose
+```
+
+Expected output on created bond is like
+
+```
+{
+  "programId": "vBoNdEvzMrSai7is21XgVYik65mqtaKXuSdMBJ1xkW4",
+  "publicKey": "...",
+  "account": {
+    "config": "vbMaRfmTCg92HWGzmd53APkMNpPnGVGZTUHwUJQkXAU",
+    "voteAccount": "...",
+    "authority": "..."
+  },
+  "voteAccount": {
+    "nodePubkey": "...",
+    "authorizedWithdrawer": "...",
+    "commission": 0
+  },
+  "amountOwned": "10.407 SOLs",
+  "amountActive": "10.407 SOLs",
+  "numberActiveStakeAccounts": 0,
+  "amountAtSettlements": "0 SOLs",
+  "numberSettlementStakeAccounts": 0,
+  "amountToWithdraw": "0 SOL",
```

Comment:
(very minor note) The `SOLs` above and `SOL` here caught my attention, it's probably just a typo here in readme and the formatting is consistent in reality, flagging.

Reply:
That's not "correct", you are right. Thanks for pointing that out. The formatting checks if the number is 0 and if so it is printed just `SOL`.
https://github.com/marinade-finance/validator-bonds/blob/main/packages/validator-bonds-cli/src/commands/utils.ts#L329-L333